deep-security-agent/defaults/main.yml

```diff
@@ -0,0 +1,41 @@
+# Copyright:: 2018, Trend Micro
+#
+# License as per [repo](master/LICENSE)
+#
+
+#######################
+# Mandatory Variables #
+#######################
+---
+# As you add agents to Deep Security, they are automatically made available for download through
+#   the same hostname:port as the Deep Security Manager admin interface. For Marketplace and software deployments, pass
+#   this data through the 'dsm_agent_download_hostname' & 'dsm_agent_download_port' attributes.
+dsa_download_hostname: app.deepsecurity.trendmicro.com
```

Comment:
The variable do not align with the comment. As the original variable do not follow the same naming rule, I would suggest to use the same set as in chef now.

Reply:
That is dsm_agent_download_hostname & dsm_agent_download_port

Reply:
Align all variables with Chef's.